themes/repair-cafe/config/fields.yaml

```diff
@@ -72,6 +72,12 @@ tabs:
             type: text
             default: 'https://plus.google.com/+KonsumentenschutzCh'
             tab: 'liip.repaircafe::lang.theme.fields.tab.general_fr'
+        google_analytics_tracking_id:
+            label: 'liip.repaircafe::lang.theme.fields.google_analytics_tracking_id'
+            span: full
+            type: text
+            default: 'UA-105831527-1'
```

Comment:
Is it possible to keep the tracking-id outside of our code like we do with our api-keys?

Reply:
I see your point but since this is just the default value which can be overridden via this field we either should remove all repair-cafe specific default values in this fields or keep all of them.

Reply:
I am not sure about this one. If the tracking-ID is nothing sensitive I would leave it like it is. Otherwise I would actually leave it empty.

Reply:
Ok I'll remove it. We just need to think about saving it in the settings after the final deployment.